src/Common/src/System/Net/Http/Http2/CopyToAspNetCore.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
* in OS like FreeBSD, bash is not available in this location in default settings, therefore most (if not all) scripts in dotnet repos use `#!/usr/bin/env bash`, i.e. a more portable shebang.
* in this script, there is no particular need for bash (mostly used for string replacement, arrays etc. facilities); so we can  switch to even more portable POSIX shell syntax: `#!/usr/bin/env sh` and replace `if [[ .. ]]` with `if [ .. ]`.

Reply:
Thanks 👍

Reply:
Scripts in corefx use bash. For the sake of consistency I'd argue it should be calling bash here as well (even if it doesn't -currently- use bash-specific constructs).

Reply:
> Scripts in corefx use bash

Yep, that's true -> https://github.com/dotnet/corefx/search?p=5&q=bash&unscoped_q=bash
So I'll change this to `#!/usr/bin/env bash` as everywhere else.